test/python/test_nixl_api.py

```
@@ -127,6 +127,44 @@ def test_nixl_conf_bad_sync_mode():
+def test_ucx_error_handling_mode_config(monkeypatch, configured_mode, expected_mode):
+    captured: list[tuple[str, dict[str, str]]] = []
+
+    class FakeAgent:
+        def getAvailPlugins(self) -> list[str]:
+            return ["UCX"]
+
+        def createBackend(self, backend: str, init_params: dict[str, str]) -> int:
+            captured.append((backend, init_params.copy()))
+            return 0
+
+        def getBackendParams(self, backend: int) -> tuple[dict[str, str], list[str]]:
```

Ruff (0.15.15)

[warning] Unused method argument: `backend`

(ARG002)